Assets/Scripts/UI/Text/TextController.cs

```diff
@@ -29,8 +41,32 @@ public void SetText(TextTypes type, string text)
         switch (type) 
         {
             case TextTypes.timer: timerText.text = text; break;
-            case TextTypes.position: positionText.text = text; break;
+            case TextTypes.position: 
+                switch (text) 
+                {
+                    case "1": text += "st"; break;
+                    case "2": text += "nd"; break;
+                    case "3": text += "rd"; break;
+                    default: text += "th"; break;
```

Comment:
Wow gamebreaking bug if we're in position 21 it'll display as 21th (jk please don't care)

Reply:
I mean ur right but when do we have 21 racers in the race 😂

Reply:
Exactly, hence the jk XD